src/OpenLoco/Scenario.cpp

```diff
@@ -24,10 +31,84 @@ namespace OpenLoco::Scenario
     static loco_global<uint8_t, 0x00526240> objectiveTimeLimitYears;
     static loco_global<uint16_t, 0x00526241> objectiveTimeLimitUntilYear;
 
+    static void sub_46115C()
+    {
+        call(0x0046115C);
+    }
+
+    static void sub_4C4BC0()
+    {
+        call(0x004C4BC0);
+    }
+
+    static void sub_49685C(int16_t year)
+    {
+        registers regs{};
+        regs.eax = year;
+        call(0x0049685C, regs);
+    }
+
+    static void sub_496A18()
+    {
+        call(0x00496A18);
+    }
+
+    static void sub_475988()
+    {
+        call(0x00475988);
+    }
+
+    static void sub_4A8810()
+    {
+        call(0x004A8810);
+    }
+
+    // TODO: Move to OrderManager::reset
+    static void sub_4702EC()
+    {
+        call(0x004702EC);
+    }
+
+    // TODO: Move to Terraform::reset
+    static void sub_4BAEC4()
+    {
+        call(0x004BAEC4);
+    }
+
+    static void sub_43C8FD()
+    {
+        call(0x0043C8FD);
+    }
+
     // 0x0043C88C
     void reset()
     {
-        call(0x0043C88C);
+        WindowManager::closeConstructionWindows();
+
+        CompanyManager::updatingCompanyId(0x0F);
+        WindowManager::setCurrentRotation(0);
+
+        CompanyManager::reset();
+        StringManager::reset();
+        ThingManager::reset();
+
+        Ui::Windows::Construction::Construction::reset();
+        sub_46115C();
+        sub_4C4BC0();
+
+        sub_49685C(0x76C);
```

Comment:
This is changing the year.
```suggestion
        sub_49685C(1900);
```